docs/UserGuide.md

```diff
@@ -599,8 +607,8 @@ Invalid usages
     * *Error message: No tutorial class with the name TUTORIAL is found*
 * Marking attendance of a tutorial with no students enrolled
     * *Error message: No students are enrolled in TUTORIAL tutorial*
-* Marking attendance of a tutorial where **all students** already has attendance marked for the corresponding week
-    * *Error message: All students in TUTORIAL tutorial has attendance marked 
+* Marking attendance of a tutorial where **all students** already have attendance marked for the corresponding week
+    * *Error message: All students in TUTORIAL tutorial have attendance marked 
```

Comment:
I think this part we need to update the error message in MarkAttendanceByTutorialCommand also right? Is the message there is still "has"?

Reply:
Yup, updated it as shown below